src/pgzero/screen.py

```diff
@@ -25,6 +31,63 @@ def make_color(arg):
     return tuple(pygame.Color(arg))
 
 
+def _get_platform_screenshot_path():
+    r"""Get the screenshot directory for pgzero.
+    Under Windows, this is %USERPROFILE%\Pictures\pgzero.
+    Under Linux/MacOS, it's ~/Pictures/pgzero.
+    If the platform is unsupported, it defaults to the CWD."""
+    if platform == "win32":
+        try:
+            home = os.environ["USERPROFILE"]
+        except KeyError:
+            raise KeyError("Couldn't find the user home directory for "
+                           "screenshots. Please set the %USERPROFILE% "
+                           "environment variable.")
+        return os.path.join(home, "Pictures", "pgzero")
+    elif platform in ["linux", "linux2", "darwin"]:
```

Comment:
I'm about 85% sure there is a peephole optimisation for this exact case in CPython, but lists must be allocated at runtime while tuples are interned in the code constants and reused, so for constant sequence literals you should use tuples:

```suggestion
    elif platform in ("linux", "linux2", "darwin"):
```

Reply:
Fixed.